components/ai_chat/core/browser/constants.cc

```diff
@@ -290,4 +291,7 @@ const base::fixed_flat_set<std::string_view, 1> kPrintPreviewRetrievalHosts =
                                                  "docs.google.com",
                                              });
 
+const char kLeoModelSupportUrl[] =
+    "https://support.brave.com/hc/en-us/categories/"
```

Comment:
Will it be a problem that we're hardcoding the `en-us` locale?

Reply:
asfik, we don't have Leo support page in other locales yet so this should be ok.